WalletWasabi.Fluent/Views/CoinControl/Core/Cells/LabelsCellView.axaml

```diff
@@ -0,0 +1,39 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:cells="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinControl.Core.Cells"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="WalletWasabi.Fluent.Views.CoinControl.Core.Cells.LabelsCellView"
+             x:DataType="cells:LabelsCellViewModel"
+             x:CompileBindings="True">
+
+  <UserControl.Resources>
+    <SolidColorBrush x:Key="PrivacyLevelNoneBrush" Color="{DynamicResource TextForegroundColor}" />
+  </UserControl.Resources>
+
+  <UserControl.Styles>
+
+    <Style Selector="controls|LabelsItemsPresenter.NonPrivate">
+      <Setter Property="BorderBrush" Value="{DynamicResource PrivacyLevelNoneBrush}" />
+      <Setter Property="Foreground" Value="{DynamicResource PrivacyLevelNoneBrush}" />
+    </Style>
+    <Style Selector="controls|LabelsItemsPresenter.SemiPrivate">
+      <Setter Property="BorderBrush" Value="{DynamicResource PrivacyLevelMediumBrush}" />
+      <Setter Property="Foreground" Value="{DynamicResource PrivacyLevelMediumBrush}" />
+      <Setter Property="MaxLabelWidth" Value="1000" />
+    </Style>
+    <Style Selector="controls|LabelsItemsPresenter.Private">
+      <Setter Property="BorderBrush" Value="{DynamicResource PrivacyLevelStrongBrush }" />
+      <Setter Property="Foreground" Value="{DynamicResource PrivacyLevelStrongBrush }" />
+      <Setter Property="MaxLabelWidth" Value="1000" />
```

Comment:
Same as above: Why we want such long labels here, wouldn't it cause layout issues in TDG ?

Reply:
The problem here is that Labels have a maximum width of 120, but there are labels that should never collapse.

Removing this, we will get this undesired behavior:

![image](https://user-images.githubusercontent.com/3109851/199504271-4df45ca5-dc97-4be8-ac72-3fc2e8048524.png)

I'd like to reset, via style, a value high enough to never collapse, so 1000 is OK here. It's an arbitrary value. It won't break TDG because this will only happen for a restricted set of labels (known pocket names).

Why I didn't set it to `double.PositiveInfinity`? Because there's easy way to set this value inside XAML :)